packages/clay-date-picker/src/index.tsx

```diff
@@ -86,6 +87,11 @@ interface IProps {
 	 */
 	spritemap?: string;
 
+	/**
+	 *
+	 */
+	timeFormat?: string;
```

Comment:
Just one caveat about adding this API: I had removed this implementation because our TimePicker only behaves with an `H:m` format, if the user escapes from that format it will not make any difference in the visualization in TimePicker maybe just in the DatePicker input here, but still in the end we’ll always be adding a `H:m` format. See `useCurrentTime`

Reply:
I don't know how it works with different formats...

Reply:
Oh good catch, I will just remove this API since we don't necessarily need it right now.